apps/sync/test/unit/TestAdapterMock.js

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+/* exported
+  TestAdapterMock
+*/
+
+const TestAdapterMock = {
+  whatAmI: 'the test adapter',
```

Comment:
Right, I thought Chai `.deep.equal` would need it, but the functions already make the object unique enough. I'm removing it.